src/includes/getting-started-config/ruby.mdx

```diff
@@ -1,7 +1,7 @@
 Initialize the Sentry SDK early in your application's setup:
```

Comment:
Note that this line is redundant to the common template. I'd suggest removing it.